### PR TITLE
Add init files for all OSes for st2timersengine [ci skip]

### PR DESCRIPTION
**Pull request by lakshmi-kannan**

### CODE REVIEW

File: packages/st2/Makefile [hunk 1]
```diff
@@ -48,12 +48,25 @@ post_install:
 	install -m644 ../conf/logrotate.conf $(DESTDIR)/etc/logrotate.d/st2
 	install -m644 ../st2actions/conf/logging.conf $(DESTDIR)/etc/st2/logging.actionrunner.conf
 	install -m644 ../st2actions/conf/syslog.conf $(DESTDIR)/etc/st2/syslog.actionrunner.conf
+	install -m644 ../st2actions/conf/syslog.conf $(DESTDIR)/etc/st2/syslog.actionrunner.conf
```
Comment (armab) [inline packages/st2/Makefile]: Dup with the previous line

**Comment by armab** [comment]
Including a new service in smoketests is needed as well, see: 
https://github.com/StackStorm/st2-packages/blob/master/rake/spec/spec_helper.rb#L24-L25

File: circle.yml [hunk 2]
```diff
@@ -19,6 +19,7 @@ jobs:
     environment:
       DISTROS: "trusty xenial el6 el7"
       ST2_PACKAGES: "st2 st2mistral"
+      ST2_GITREV: "k8s/split_timers"
```
Comment (lakshmi-kannan) [inline circle.yml]: Fix this before mege.

**Comment by Kami** [comment]
I will merge this due to the e2e test failing issue described in https://github.com/StackStorm/st2/pull/4180#issuecomment-411064458.

And once the tests pass there and that st2 PR is merged, I will update `ST2_GITREV` in this repo.

**Pull request merged**